molgenis-data/src/main/java/org/molgenis/data/meta/model/AttributeMetadata.java

```diff
@@ -160,6 +165,7 @@ public void init()
 		addAttribute(IS_READ_ONLY).setDataType(BOOL).setNillable(false).setLabel("Read-only");
 		addAttribute(IS_UNIQUE).setDataType(BOOL).setNillable(false).setLabel("Unique");
 		addAttribute(TAGS).setDataType(MREF).setRefEntity(tagMetadata).setLabel("Tags");
+		addAttribute(NULLABLE_EXPRESSION).setDataType(SCRIPT).setNillable(true).setLabel("Nullable expression");
```

Comment:
breaking change -> upgrade version number to 6.0.0-SNAPSHOT

Reply:
lets do this in a separate pull request somewhere before the release.